.release-notes/3992.md

```diff
@@ -0,0 +1,12 @@
+## `itertools` additions
+
+Previously there were methods for `Iter` within `itertools` which, while originally planned, where not implemented. These methods have now been implemented.
+
+Added methods include:
```

Comment:
"include" suggests there are others. Are there others? If not, this should be, "The added methods are:"